mmv1/products/firebaseappcheck/firebase_app_check_service_config.yaml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ServiceConfig'
+base_url: projects/{{project}}/services/
+self_link: projects/{{project}}/services/{{service_id}}
+create_verb: :PATCH
+update_verb: :PATCH
+update_mask: true
+delete_verb: :PATCH
```

Comment:
We don't support deletion at all for this resource, in case that matters.

Reply:
Here, deletion will turn it OFF. I think this is the right behavior because OFF is the state before "creation". Another option is to treat deletion as a no-op, leaving the enforcement state as-is, and allowing an explicit OFF mode. WDYT? @weixifan

Reply:
Yes, that would work. Out of curiosity, which file contains the logic that "deletion will turn it OFF"?

Reply:
With those two lines:
```yaml
delete_url: projects/{{project}}/services/{{service_id}}?updateMask=enforcementMode
delete_verb: :PATCH
```
and an absence of `custom_delete` in the `custom_code` section, the deletion request will be

```
PATCH firebaseappcheck.googleapis.com/{{project}}/services/{{service_id}}?updateMask=enforcementMode
{} (empty body)
```

Which is essentially turning it `OFF`.

Reply:
This discussion thread looks resolved to me, but I'll wait for an ack from one of you before I click merge.

Reply:
Yes this is good to merge. Leaving this block open so @weixifan can find it.